comp/otelcol/otlp/components/exporter/datadogexporter/go.mod

```diff
@@ -102,19 +102,14 @@ require (
 	github.com/DataDog/datadog-agent/pkg/proto v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/serializer v0.56.0-rc.3
 	github.com/DataDog/datadog-agent/pkg/trace v0.56.0-rc.3
-	github.com/DataDog/datadog-agent/pkg/util/hostname/validate v0.57.1
 	github.com/DataDog/datadog-go/v5 v5.5.0
 	github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/attributes v0.21.0
 	github.com/DataDog/opentelemetry-mapping-go/pkg/otlp/metrics v0.21.0
+	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/datadog v0.111.0
```

Comment:
why are we importing v0.111.0 when current collector deps are on v0.113.0?

Reply:
Good catch, it's a bad merge, when I started this PR the converged agent still uses 0.111